### ISSUE TITLE
[playground] Infura & Etherscan load balancing leads to out-of-sync state

### ISSUE THREAD

**Issue by snario** [counterfactual/monorepo#998]
This leads to one Node correctly proposing a state transition for a `view` function (important for the `deposit` method on the Node) but the other Node not seeing it as valid and thus the `uninstall` protocol will fail. We need something more reliable than these APIs.

**Comment by emansipater** [comment]
Shouldn't it also just keep rechecking though?

**Comment by snario** [comment]
@emansipater Yeah, we're going to add this as a fix but the API _should_ be more reliable still.

**Comment by snario** [comment]
Added in #1007
